data/mods/Aftershock/mobs/aliens.json

```diff
@@ -0,0 +1,155 @@
+[
+  {
+    "id": "afs_mon_rabbit",
+    "type": "MONSTER",
+    "name": { "str": "labbit" },
+    "description": "A rabbit from earth, genetically modified to eat the lichen of Salus IV.  It's fur is rust colored and its ears are shortened to preserve warmth.",
+    "default_faction": "small_animal",
+    "bodytype": "pig",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "1600 ml",
+    "weight": "1600 g",
+    "hp": 8,
+    "speed": 160,
+    "material": [ "flesh" ],
+    "symbol": "r",
+    "color": "brown",
+    "aggression": -99,
+    "morale": -7,
+    "melee_cut": 0,
+    "dodge": 6,
+    "reproduction": { "baby_monster": "afs_mon_rabbit", "baby_count": 3, "baby_timer": 55 },
+    "harvest": "alien_small_fur",
```

Comment:
If it was genetically modified would it still be as poisonous to eat, compared to local fauna?

Reply:
The idea is that its sustaining off the local flora in the area, and thus is modified to metabolize the toxic macronutrient, so it would be poisonous yes

Reply:
If it metabolizes the toxic substance, and the toxin is a macromolecule - not something like a heavy metal - then that should remove said toxin. If it's simply tolerant to the toxin, that's another matter.

Reply:
I'm not sure as to why it would be brought if it didn't clear the toxin - just for the fur?

Reply:
Keep in mind these genetically modified animals are from when the colony first set up hundreds of years ago, and then collapsed. 

They probably either brought it as a pet, and the modification was to make feeding it easier, or they weren't aware of the toxin, or it could've arised later.

Reply:
Ah! (Is there anyplace this backstory is codified?)

Reply:
BTW, if you're interested, some of the backstory (especially stuff PCs would know) could be put on the CDDA wiki, I think.

Reply:
This is for new lore, and its still all WIP right now, so its not codified yet anywhere